### DISCUSSION TITLE
M2O simple pulldown

### DISCUSSION THREAD

**Discussion by x1B7F** [directus/directus#8124] [Archive]
I built something a year ago using Directus with a M2O and it was a simple pulldown at the time. Came back to do a similar project and there have been lots of updates to Directus since I last used it. Can you no longer have just a simple HTML pulldown dialogue for a many to one?

The new selector visually looks sweet but for my use the old pulldown was more effective and saved a step. Now you need to select the item in the list then also save your selection with a second step.

Cheers

**Comment by rijkvanzanten** [comment]
> Can you no longer have just a simple HTML pulldown dialogue for a many to one?

This is correct. All dropdowns in the app use the custom styled dropdowns

**Reply by x1B7F** [reply]
Thanks
